content/apidokumentasjon/reelle-rettighetshavere/maskinell-innrapportering/beskrivelse-av-felter/_index.md

```diff
@@ -4,6 +4,66 @@ description: Tekstlig beskrivelse av feltene i JSON-skjemaet
+|                                                                        | reelleRettighetshavereidentifikasjon                                                                                                                | Kodeverdi (fra kodelista reelleRettighetshavereidentifikasjon) som angir om virksomheten har / kan identifisere reelle rettighetshavere. Angi en av 3 følgende verdier:  1. Har reelle rettighetshavere `reelleRettighetshavereidentifikasjon.harReelleRettighetshavere`  2. Har reelle rettighetshavere, men ingen av disse kan identifiseres `reelleRettighetshavereidentifikasjon.ingenReelleRettighetshavereKanIdentifiseres`  3. Har ikke reelle rettighetshavere `reelleRettighetshavereidentifikasjon.harIkkeReelleRettighetshavere` | Ja          | Ja       |                                                                                                                                                                                                                                                                   |
+|                                                                        | aarsakTilAtVirksomhetenIkkeHarReelleRettighetshavere.erEidEllerKontrollertAvOffentligVirksomhet                                                     | Boolsk verdi som angir om årsaken til at virksomheten ikke har reelle rettighetshavere er at den er kontrollert av en offentlig myndighet                                                                                                                                                                                                                                                                                                                                                                                                   | Ja          | Betinget | Må angis hvis reelleRettighetshavereidentifikasjon er satt til:  "reelleRettighetshavereidentifikasjon.harIkkeReelleRettighetshavere"                                                                                                                              |
```

Comment:
Fjern en space helt til høyre i tabellen for property  aarsakTilAtVirksomhetenIkkeHarReelleRettighetshavere.erEidEllerKontrollertAvOffentligVirksomhet  slik at tabellen får riktig "alignment"